### ISSUE TITLE
db_connect does not inform user of db_disconnect requirement

### ISSUE THREAD

**Issue by shuckins-r7** [rapid7/metasploit-framework#4868]
If you are already connected to a DB, the db_connect command won't actually connect. Output of the command, as well as help text, do not indicate this to the user in any way. IMO we should disconnect first when connect is run. Prompt the user to confirm for extra kindness points.

## Steps to reproduce
```
msfpro
```

## Connection string
```
[*] Successfully loaded plugin: pro
msf-pro > db_status
[*] postgresql connected to metasploit_credentials
msf-pro > db_connect msf:pass123@localhost:5432/metasploit_pro_master
msf-pro > db_status
[*] postgresql connected to metasploit_credentials
```

### yaml
```
$ cat /tmp/config.yml
production:
  adapter: postgresql
  username: msf
  password: pass123
  host: localhost
  port: 5432
  pool: 50
  timeout: 5
  database: metasploit_credentials
msf-pro > db_status
[*] postgresql connected to metasploit_pro_master
msf-pro > db_connect -y /tmp/config.yml
msf-pro > db_status
[*] postgresql connected to metasploit_pro_master
```

### Help text, help more!
```
msf-pro > help db_connect
[*]    Usage: db_connect <user:pass>@<host:port>/<database>
[*]       OR: db_connect -y [path/to/database.yml]
[*] Examples:
[*]        db_connect user@metasploit3
[*]        db_connect user:pass@192.168.0.2/metasploit3
[*]        db_connect user:pass@192.168.0.2:1500/metasploit3
```

**Comment by wvu-r7** [comment]
[![Help computer](https://img.youtube.com/vi/UE_czPc7o5Y/0.jpg)](https://www.youtube.com/watch?v=UE_czPc7o5Y)

Help computer.

**Comment by wvu-r7** [comment]
Fixed in #4868.

```
msf > db_connect 
[-] postgresql already connected to metasploit_framework_development
[-] Run db_disconnect first if you wish to connect to a different database
```

**Comment by XSecr3t** [comment]
Excuse me

My db not connection
Every i active postgresql always 
No connection selection

I check database.yml on
data/data/com.termux/files/home/metasploit-framework/config/database.yml
And i check i see this. 

> adapter: postgresql
                            database: msf_database
                            username: msf
                            password:
                            host: 127.0.0.1
                            port: 5432
                            pool: 75
                            timeout: 5

Is this mean something?  I think, postgresql should be have passwords and default port.
